docs/modding.md

```diff
@@ -78,3 +78,26 @@ See the example mod to learn about some of the properties that different types o
 //TODO help wanted!
 
 #### Items & Other Content
+
+Every item needs also their properties:
+-It's `type`. For items there are 2 (technically 3, if you consider Liquids): Material and Resource. The difference between the two is that Core accepts Materials, but not resources. Liquids are "resources" what require conduits, and act very differently.
```

Comment:
> (technically 3, if you consider Liquids)

I don't know what you mean. Items aren't related to liquids at all, even by type.